backend/db_migrations/winning-losing-rating-change.sql

```diff
@@ -0,0 +1,7 @@
+ALTER TABLE "Matches" RENAME COLUMN "RatingChange" TO "WinningTeamRatingChange";
```

Comment:
i don't know how we're running the migrations, BUT
1. don't we want to order the migration file names?

0-setup-database.sql
1.-winning-losing-rating-change.sql
2.--- 
?

2. how about mentioning it in the readme?

Reply:
I think we should order them at the very least. I can rename the files.

What exactly do you want to be mentioned in the readme?

Reply:
anything that would help the uninitiated run the app. just mention it in the setup phase, that they should run the migrations